week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -4,7 +4,8 @@
 function mood() {
   let isHappy = true;
 
-  if (isHappy) {
+  // if (mood === isHappy) {
+    if (!isHappy) {
```

Comment:
This does pass the test, but it seems a bit counter-intuitive now. The function returns `"I am not happy"` when `isHappy === true` and it returns `"I am happy"` when `isHappy === false`. That might be quite confusing and when writing code, it is important to be as clear as possible. 

Could you think of a way to make the test pass without changing the `if` statement?